lib/puppet_fixtures.rb

```diff
@@ -452,9 +450,7 @@ def download(flags = nil, subdir = nil)
           update
         else
           clone(flags)
-          unless File.exist?(@target)
-            raise "Failed to clone repository #{@remote} into #{@target}"
-          end
+          raise "Failed to clone repository #{@remote} into #{@target}" unless File.exist?(@target)
```

Comment:
I hate the guard clause cop if I'm honest

Reply:
I disabled the cop